masu/database/report_stats_db_accessor.py

```diff
@@ -0,0 +1,215 @@
+        """
+        Getter for last_started_datetime.
+
+        Args:
+            None
+        Returns:
+            (DateTime): Time stamp for last started date/time.
+        """
+        obj = self._get_db_obj_query().first()
+        return obj.last_started_datetime
+
+    def log_last_started_datetime(self):
+        """
+        Convinence method for logging start processing.
+
+        Args:
+            None
+        Returns:
+            None
+        """
+        self.update(last_started_datetime=datetime.now())
+
+    def log_last_completed_datetime(self):
+        """
+        Convinence method for logging processing completed.
+
+        Args:
+            None
+        Returns:
+            None
+        """
+        self.update(last_completed_datetime=datetime.now())
+
+    def get_etag(self):
+        """
+        Getter for the report file's etag.
+
+        Args:
+            None
+        Returns:
+            (String): MD5 hash of object.
+        """
+        obj = self._get_db_obj_query().first()
+        return obj.etag
+
+    def add(self, fields_dict):
+        """
+        Add a new row to the CUR stats database.
+
+        Args:
+            (Dictionary): Fields containing CUR Status attributes.
+
+            Valid keys are: report_name,
+                            cursor_position,
+                            last_completed_date (optional),
+                            last_started_date (optional),
+                            etag (optional)
+        Returns:
+            None
+
+        """
+        new_entry = self._costentrystatus(**fields_dict)
+        self._session.add(new_entry)
+
+    def remove(self):
+        """
+        Remove a CUR statistics from the database.
+
+        Args:
+            None
+        Returns:
+            None
+        """
+        obj_to_remove = self._get_db_obj_query().first()
+        self._session.delete(obj_to_remove)
+
+    def update(self,
+               cursor_position=None,
+               last_completed_datetime=None,
+               last_started_datetime=None,
+               etag=None):
+        """
+        Update a CUR statistics record in the database.
+
+        Args:
+            (Dictionary): Fields containing CUR Status attributes.
+
+            Valid keys are: cursor_position,
+                            last_completed_date,
+                            last_started_date,
+                            etag
+        Returns:
+            None
+        """
+        obj_to_update = self._get_db_obj_query()
+        update_dict = {}
+        if cursor_position:
+            update_dict['cursor_position'] = cursor_position
+        if last_completed_datetime:
+            update_dict['last_completed_datetime'] = last_completed_datetime
+        if last_started_datetime:
+            update_dict['last_started_datetime'] = last_started_datetime
+        if etag:
+            update_dict['etag'] = etag
+
+        obj_to_update.update(update_dict)
```

Comment:
Same comment as above, instead of using the query's update method, I think we should just modify the object's attributes directly and then you can just commit the session to save. 

```
if cursor_position:
    obj.cursor_position = cursor_position
...

self.commit()
```